helm/defectdojo/templates/pvc.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.django.uwsgi.persistentVolume.enabled -}}
+{{- if not .Values.django.uwsgi.persistentVolume.existingClaim -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    "helm.sh/resource-policy": keep
+  name: {{ include "defectdojo.fullname" . }}-media
+spec:
+  accessModes:
+  - ReadWriteOnce
+{{- with .Values.django.uwsgi.persistentVolume.storageClass }}
```

Comment:
Why this when .Values.django.uwsgi.persistentVolume.storageClass is not list or array...

Reply:
Changed to storageClassName: "{{ .Values.django.uwsgi.persistentVolume.storageClass }}